_pages/en/capabilities.md

```diff
@@ -505,27 +492,42 @@ InfoSec supplies development teams and operational teams with an ample number of
 #### 3.3.2. Indicators  
 
 - Are people in your team often assigned to work on multiple tasks ? If yes , what is the main reason?  
```

Comment:
Two questions in one?

- Are people in your team often assigned to work on multiple tasks ? If yes , what is the main reason?  
==>

People on my team are often assigned to work on multiple mutually exclusive tasks
- Strongly agree
- Ideal: Strong disagree

Reply:
Or remove it, because we already have... 

- In my team we do not require people to split their time between multiple tasks